tests/introspection/tests.py

```diff
@@ -152,22 +150,6 @@ def test_get_relations(self):
             editor.add_field(Article, body)
         self.assertEqual(relations, expected_relations)
 
-    @skipUnless(connection.vendor == 'sqlite', "This is an sqlite-specific issue")
-    def test_get_relations_alt_format(self):
-        """
-        With SQLite, foreign keys can be added with different syntaxes and
-        formatting.
-        """
-        create_table_statements = [
-            "CREATE TABLE track(id, art_id INTEGER, FOREIGN KEY(art_id) REFERENCES {}(id));",
-            "CREATE TABLE track(id, art_id INTEGER, FOREIGN KEY (art_id) REFERENCES {}(id));"
-        ]
-        for statement in create_table_statements:
-            with connection.cursor() as cursor:
-                cursor.fetchone = mock.Mock(return_value=[statement.format(Article._meta.db_table), 'table'])
-                relations = connection.introspection.get_relations(cursor, 'mocked_table')
-            self.assertEqual(relations, {'art_id': ('id', Article._meta.db_table)})
-
```

Comment:
Figured there was not much value in keeping this test around anymore with all the parsing logic gone but here's an adapted version you think otherwise

```python
@skipUnless(connection.vendor == 'sqlite', "This is an sqlite-specific issue")
def test_get_relations_alt_format(self):
    """
    With SQLite, foreign keys can be added with different syntaxes and
    formatting.
    """
    with connection.cursor() as cursor:
        cursor.execute('CREATE TABLE introspection_alt_fk (id, art_id INTEGER, FOREIGN KEY(art_id) REFERENCES introspection_alt_fk(id))')
        relations = connection.introspection.get_relations(cursor, 'introspection_alt_fk')
        cursor.execute('DROP TABLE introspection_alt_fk')
    self.assertEqual(relations, {'art_id': ('id', 'introspection_alt_fk')})
```